code/game/objects/items/latexballoon.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/latexballon
 	name = "latex glove"
-	desc = "" //todo
+	desc = "Sterile and airtight."

```

Comment:
Like seriously, the description could of been "A latex balloon." and IT STILL WOULD OF BEEN BETTER.